src/token/history/dto/item-token.response.ts

```diff
@@ -0,0 +1,11 @@
+import { Field, Int, ObjectType } from '@nestjs/graphql'
+import { Token } from '../../entities/token.entity'
+
+@ObjectType()
+export class ItemToken {
```

Comment:
You create it but you didn't use it. and it is not for TokenHistory. You must use it for getAllToken